var/spack/repos/builtin/packages/nut/package.py

```diff
@@ -24,11 +24,12 @@ class Nut(CMakePackage):
     depends_on('cmake@3.0:', type='build')
     depends_on('random123')
 
+    conflicts('%nvhpc)
```

Comment:
```suggestion
    # The conflict with %nvhpc is inherited from random123,
    # which is a C++ template library
    conflicts('%nvhpc)
```

Reply:
My bad, I didn't make it clear I hadn't tested the code:
```suggestion
    conflicts('%nvhpc')
```

Reply:
commited. thanks.